.travis.yml

```diff
@@ -2,51 +2,53 @@ language: scala
 dist: xenial
 jdk: openjdk8
 if: tag IS blank
-
 branches:
   only:
   - master
-
 addons:
   apt:
     sources:
-      - sourceline: 'ppa:ubuntu-toolchain-r/test'
+    - sourceline: ppa:ubuntu-toolchain-r/test
     packages:
-      - cmake
-      - g++-8
-
+    - cmake
+    - g++-8
 env:
+  matrix:
   - CXX=/usr/bin/g++-8
```

Comment:
This is a great change - should probably consider adding Clang here, too.

Reply:
that happened automagically too by the travis gem, i guess it's not really a change but just a refactor? no idea though.